BWV 1004 Violin Partita II D minor/5-ciaccona.ly

```diff
@@ -420,7 +421,7 @@
     d,16) bes,32( c d16) f d bes d f g32( a bes c d e f g)
     
     % 71
-    cis, bes'( a g f e d \once\omit Accidental cis) bes( a g f e d cis b) a( b cis d e f g e) % 2nd cis seems to be an oversight so keeping cis but hiding accidental to match manuscript
+    cis, bes'( a g f e d c) bes( a g f e d cis b) a( b cis d e f g e) % 2nd cis seems to be an oversight so keeping cis but hiding accidental to match manuscript
```

Comment:
Just checked and multiple editors use c, so I'll take it. Please, remove the comment (at the end of the line) talking about this because it will be no longer a cis.